pkg/minikube/node/start.go

```diff
@@ -972,3 +972,8 @@ To see benchmarks checkout https://minikube.sigs.k8s.io/docs/benchmarks/cpuusage
 `, out.V{"drivers": altDriverList.String()})
 	}
 }
+
+// ValidWindowsOS lists the supported Windows OS versions
+func ValidWindowsOSVersions() []string {
+	return []string{"2019", "2022"}
```

Comment:
```suggestion
func ValidWindowsOSVersions() map[string]bool {
	return map[string]bool{"2019": true, "2022": true}
```

Would be easier to check this as a map

Reply:
for some reason the unit test is failing with this as a map. I am not sure the return map will always have 2019 first then 2022. === test failure message -> Invalid Windows Server OS Version: 2023. Valid OS version are: [2019 2022], expected Invalid Windows Server OS Version: 2023. Valid OS version are: [2019 2022]

Reply:
You just need to add the `maps` to the import at the top of the file.

Reply:
I see, yeah that will be a problem. The easiest would likely be to change it back to a `[]string`, but you can simplify the loop you had before, if `option` equals the version you're looking for you can just return `nil`. Another option is you could sort the slice before outputting it with `slices.Sort` which would also resolve your issue, but the former is likely a better option. The overkill option if you make a `type` (`WindowsOSVersions`) and you add a `String()` to it which does the `maps.Keys` and `slices.Sort` but is probably a bit overkill for what you need.